presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -552,7 +561,23 @@ private PlanBuilder aggregate(PlanBuilder subPlan, QuerySpecification node)
                     argumentMarkers.put(args, marker.get());
                 }
             }
-            aggregations.put(newSymbol, new Aggregation((FunctionCall) rewritten, analysis.getFunctionSignature(aggregate), marker));
+
+            aggregations.put(newSymbol, new Aggregation(
+                    (FunctionCall) rewritten,
+                    analysis.getFunctionSignature(aggregate),
+                    marker,
+                    aggregate.getSortItems().map(
+                            sortItems -> sortItems.stream()
+                                    .map(SortItem::getSortKey)
+                                    .map(sortKey -> ExpressionTreeRewriter.rewriteWith(new ParameterRewriter(analysis.getParameters(), analysis), sortKey))
```

Comment:
Sort keys can only be references to columns, while parameters are constants. It doesn't seem that one can use parameter as order by argument of aggregation, so the rewrite is unnecessary